modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -91,20 +91,37 @@ function {{classname}}:{{operationId}}({{#allParams}}{{#required}}{{paramName}}{
         return nil, stream, errno
     end
     local http_status = headers:get(":status")
-    if http_status == "200" then
+    if tonumber(http_status) >= 200 or tonumber(http_status) <= 299 then
```

Comment:
No need to convert to number (and that will have false positives, e.g. a status code of `2e2`)
Instead consider if you just want to check the first character? In which case: `if http_status:sub(1,1) == "2"`